packages/keystone/logging/GraphQLLoggerApp.js

```diff
@@ -74,17 +92,16 @@ class GraphQLLoggerPlugin {
             async didEncounterErrors (requestContext) {
                 const logData = getGraphQLReqLoggerContext(requestContext)
                 const errors = get(requestContext, 'errors', [])
-                const responseTime = Number(process.hrtime.bigint() - startTime) / 1000000
```

Comment:
This one is calculated somewhere else?

Reply:
Yup, total responseTime will be shown at the final event of the apollo lifecycle. didEncounterErrors called before willSendResponse, so IMO we don't need to calculate time for error throwing process